### ISSUE TITLE
Get error from page.get_pixmap() in version 1.18.18

### ISSUE THREAD

**Issue by Augus1999** [pymupdf/PyMuPDF#1266]
## Describe the bug
Get error `AttributeError: 'Pixmap' object has no attribute 'samples_mv'` when using method page.get_pixmap()

## To Reproduce
The code I used is
```python
page_pixmap = page.get_pixmap(
        matrix=fitz.Matrix(1.0, 1.0),
        clip=True,
    )
```
This worked well in old PyMuPDF versions.

## System configuration
 - Operating system : Windows 10 21H1 64-bit
 - Python version: Python 3.9.7 64-bit
 - PyMuPDF version==1.18.18 installed from wheel

**Comment by JorjMcKie** [comment]
There is indeed an error: the two attributes `Pixmap.samples_mv`, `Pixmap.samples_ptr` are not correctly set by .get_pixmap()` methods.
But the pixmap does get created in any case I tested.

**Comment by JorjMcKie** [comment]
No, sorry, this error is a bit more serious. I think I need to submti a hotfix 😒

**Comment by Augus1999** [comment]
Hi, I do find `self.samples_mv` and `self.samples_ptr` in `__init__` method of Pixmap, so I can't understand why the error occurs. I changed line 6791 of fitz.py into `return bytes(self._samples_mv())` and Pixmap.samples works 😶.

**Comment by JorjMcKie** [comment]
Well, I do understand - a stupid error.
I need to do these assignments as well, when pixmaps get created for pages, annotations and DisplayLists, which I forgot.
So the result of ``page.get_pixmap()``does have method `pix._samples_mv()`, but the assignment `pix.samples_mv = pix._samples_mv()` was not made.
Because this is the most frequent way of creatng pixmaps, I do need to publish a hotfix ...

**Comment by JorjMcKie** [comment]
> I changed line 6791 of fitz.py into return bytes(self._samples_mv()) and Pixmap.samples works 😶.

good job btw

**Comment by JorjMcKie** [comment]
Just published v1.18.19 to PyPI.
